app/controllers/classification_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ClassificationController < AuthenticateController
+  def create
+    pockets = Pocket.where(id: classification_params[:pocket_ids], organization: logged_user.organization)
+    return render_error(1, 'Negative kg of trash') if classification_params[:kg_trash].negative?
```

Comment:
i think we should change this validations to the model

Reply:
i saw you already added this numerical validations to the model so you should remove this checks